django/shortcuts.py

```diff
@@ -96,6 +83,9 @@ def get_object_or_404(klass, *args, **kwargs):
     queryset = _get_queryset(klass)
     try:
         return queryset.get(*args, **kwargs)
+    except AttributeError:
+        raise ValueError("First argument to get_object_or_404 must be a Django Model, "
```

Comment:
get_object_or_404() must be a Model, ... (add parenthesis chop "Django")
using hanging indent 
do you think including the object's type in the error message as done before isn't helpful?